src/parameterTypes.ts

```diff
@@ -0,0 +1,108 @@
+import { defineParameterType } from "@cucumber/cucumber";
+import memory from "@qavajs/memory";
+
+defineParameterType({
+  name: "memory",
+  regexp: /'(.+)'/,
+  transformer: (p: string) => memory.getValue(p),
+});
+
+/**
+ * Used for returning key code by key name
+ */
+defineParameterType({
+  regexp: /"([^"\\]*(\\.[^"\\]*)*)"/,
+  transformer(keyName: string) {
+    const KEYS: any = {
+      Enter: "Enter",
+      "Ctrl+A": process.platform === "darwin" ? ["Meta", "a"] : ["\uE009", "a"],
+      Tab: "Tab",
+      ArrowRight: "ArrowRight",
+      ArrowLeft: "ArrowLeft",
+    };
+    return KEYS[keyName];
+  },
+  name: "key",
+  useForSnippets: false,
+});
+
+/**
```

Comment:
pls change description, seems like it is for waits

Reply:
done